pkg/container-utils/containerutils.go

```diff
@@ -181,3 +181,41 @@ func GetIfacePeers(pid int) ([]*net.Interface, error) {
 
 	return ifacesHost, err
 }
+
+// GetIfaces returns the interfaces inside the container
+// It returns only veth and up devices
+func GetIfaces(pid int) ([]*net.Interface, error) {
+	var ifaces []*net.Interface
+
+	err := nsenter.NetnsEnter(pid, func() error {
+		links, err := netlink.LinkList()
+		if err != nil {
+			return fmt.Errorf("getting links: %w", err)
+		}
+
+		for _, link := range links {
+			veth, ok := link.(*netlink.Veth)
+			if !ok {
+				continue
+			}
```

Comment:
It will not work with `--host`. But at the moment, there are no tests for `--host`, so that's ok?